### PR TITLE
Update AntD ListField styles.

### LINKED ISSUE #773: Antd Styling
You're doing a really great job with v3! 🥇 

Since antd4 the border-radius is set to 2px. If it is an easy fix, I think it would look nice, if the border-radius is the same for the ListField. See:

Screenshot 2020-07-27 at 11 45 21

Thank you so much.
Oliver

### PR DESCRIPTION
**Pull request by kestarumper**
This PR closes #773 by updating the AntD styles of the `ListField` component to match the newest version.

### CODE REVIEW

File: packages/uniforms-antd/src/DateField.tsx [hunk 1]
```diff
@@ -1,19 +1,26 @@
 import DatePicker, { DatePickerProps } from 'antd/lib/date-picker';
 import React, { Ref } from 'react';
 import moment, { Moment } from 'moment';
-import { FieldProps, connectField, filterDOMProps } from 'uniforms';
+import { connectField, FieldProps, filterDOMProps } from 'uniforms';
```
Comment (radekmie) [inline packages/uniforms-antd/src/DateField.tsx:4]: Previous ordering was correct.

File: packages/uniforms-antd/src/DateField.tsx [hunk 2]
```diff
@@ -1,19 +1,26 @@
...
 
 import wrapField from './wrapField';
 
+// FIXME: Seems like TimePickerProps doesn't contain all needed types, i.e. 'showTime'
```
Comment (radekmie) [inline packages/uniforms-antd/src/DateField.tsx:8]:
```suggestion
// FIXME: Seems like DatePickerProps doesn't contain 'showTime'.
```

File: packages/uniforms-antd/src/DateField.tsx [hunk 3]
```diff
@@ -1,19 +1,26 @@
...
   DatePickerProps,
-  { inputRef?: Ref<typeof DatePicker> }
+  { inputRef?: Ref<typeof DatePicker>; showTime: boolean }
 >;
 
-function Date(props: DateFieldProps) {
+const defaultWrapperStyle = { width: '100%' };
+
+function Date({
+  showTime = true,
```
Comment (radekmie) [inline packages/uniforms-antd/src/DateField.tsx:18]: It should be used below.

File: packages/uniforms-antd/src/ListAddField.tsx [hunk 4]
```diff
@@ -30,7 +42,13 @@ function ListAdd({ disabled, name, value, ...props }: ListAddFieldProps) {
...
   return (
     <Button
-      {...filterDOMProps(props)}
+      {...filterDOMProps({
+        icon,
+        size,
+        style,
+        type,
+        ...props,
+      })}
```
Comment (radekmie) [inline packages/uniforms-antd/src/ListAddField.tsx:51]:
```suggestion
      icon={icon}
      size={size}
      style={style}
      type={type}
      {...filterDOMProps(props)}
```

File: packages/uniforms-antd/src/ListDelField.tsx [hunk 5]
```diff
@@ -31,7 +36,13 @@ function ListDel({ disabled, name, ...props }: ListDelFieldProps) {
 
   return (
     <Button
-      {...filterDOMProps(props)}
+      {...filterDOMProps({
```
Comment (radekmie) [inline packages/uniforms-antd/src/ListDelField.tsx:39]: Same as in `ListAddField`.

File: packages/uniforms-antd/src/ListField.tsx [hunk 6]
```diff
@@ -35,12 +42,17 @@ function List({
...
   value,
   wrapperCol,
   ...props
 }: ListFieldProps) {
   return (
-    <div {...filterDOMProps(props)}>
+    <div
+      {...filterDOMProps(props)}
+      style={style}
+      className={classNames([props.className, 'ant-list', 'ant-list-bordered'])}
```
Comment (radekmie) [inline packages/uniforms-antd/src/ListField.tsx:54]: Extract `className` from the `props` above.

File: packages/uniforms-antd/src/ListItemField.tsx [hunk 7]
```diff
@@ -6,34 +6,41 @@ import ListDelField from './ListDelField';
 
 export type ListItemFieldProps = { children?: ReactNode; value?: unknown };
 
-function ListItem({
-  children = <AutoField label={null} name="" />,
-}: ListItemFieldProps) {
+const listDelFieldWrapperStyle = {
+  float: 'right' as any,
```
Comment (radekmie) [inline packages/uniforms-antd/src/ListItemField.tsx:10]: Is this `any` needed?

Comment (Monteth) [reply packages/uniforms-antd/src/ListItemField.tsx:10]: > Types of property 'float' are incompatible.     Type 'string' is not assignable to type '"right" | "none" | "-moz-initial" | "inherit" | "initial" | "revert" | "unset" | "inline-end" | "inline-start" | "left" | undefined'.

Comment (radekmie) [reply packages/uniforms-antd/src/ListItemField.tsx:10]: Then use `'right' as const`.

File: packages/uniforms-antd/src/ErrorField.tsx [hunk 8]
```diff
@@ -6,26 +6,28 @@ export type ErrorFieldProps = Override<
...
+  ...props
+}: ErrorFieldProps) {
   return !error ? null : (
-    <div {...filterDOMProps(props)}>
-      {children ? (
-        children
-      ) : (
-        <div style={{ margin: '3px' }}>{errorMessage}</div>
-      )}
+    <div {...filterDOMProps({ style, ...props })}>
```
Comment (Monteth) [inline packages/uniforms-antd/src/ErrorField.tsx:27]: We can apply style on `div` without filtering.

Comment (Monteth) [inline packages/uniforms-antd/src/ErrorField.tsx:27]:
```suggestion
    <div style={style} {...filterDOMProps(props)}>
```

**Pull request merged**